subsys/bluetooth/mesh/proxy_srv.c

```diff
@@ -75,6 +78,17 @@ static struct bt_mesh_proxy_client {
 
 static bool service_registered;
 
+#if defined(CONFIG_BT_MESH_PRIV_BEACONS)
+/* Holds global node id state: private, non-private and amount of adv-ed subnets. */
```

Comment:
`Holds global node id state: private, non-private and number of adv-ed subnets.`